File: skills/avifenesh--agentsys/__tests__/fixtures/repo-map/go/sample.go
package sample

import "fmt"
import pkg "os"

const PublicConst = 1
const privateConst = 2

type PublicType struct { Name string }
type privateType interface { Run() }

func PublicFunc(value int) int { return value + 1 }
func privateFunc() { fmt.Println(pkg.PathSeparator) }
func (t PublicType) Method() int { return len(t.Name) }

File: skills/avifenesh--agentsys/__tests__/fixtures/repo-map/rust/sample.rs
use std::collections::{HashMap, HashSet};
use std::fmt;

pub struct PublicStruct {
    value: i32,
}
struct PrivateStruct {
    value: i32,
}

pub enum PublicEnum {
    One,
    Two,
}
pub trait PublicTrait {
    fn run(&self);
}

pub const PUBLIC_CONST: i32 = 1;
const PRIVATE_CONST: i32 = 2;

pub fn public_fn() -> i32 {
    1
}
fn private_fn() -> i32 {
    2
}
